static/js/pages/home.js

```diff
@@ -98,6 +98,11 @@ var app = new Vue({
         this.activePrompts = prompts.filter(p => !p.rated);
 
         this.lobbies = await getUserLobby(this.loggedIn);
+        for(let lobby of this.lobbies){
```

Comment:
If you want to modernize / functionalize this you can do something like 
```
this.hiddenLobbies = this.lobbies.reduce((count, lobby) => count + lobby.isHidden, 0);
```

Reply:
If you want to make the change and commit it feel free and then we can merge this. Here is some reading on map, reduce, and filter in JS

Reply:
Gotcha, thanks for that link too. I went ahead and made the changes suggested